tests/integration/api/test_observation_table.py

```diff
@@ -94,6 +94,8 @@ async def test_observation_table_from_source_table(
     check_materialized_table_preview_methods(
         observation_table, expected_columns=["POINT_IN_TIME", "User ID"]
     )
+    assert observation_table.earliest_point_in_time is not None
+    assert "User ID" in observation_table.entity_column_name_to_count
```

Comment:
We could make the check stronger by downloading the observation table as a dataframe and check that the stats are expected. Something like:

```python
df = observation_table.to_pandas()
expected_min = df["POINT_IN_TIME"].min()
expected_max = df["POINT_IN_TIME"].max()
for col in entity_cols:
    entity_unique_count = df[col].nunique()
```